### PR TITLE
🏗 Remove esmPassCompilation from build-system

### PR DESCRIPTION
**Pull request by jridgewell**
The ESM output is identical with and without it.

### CODE REVIEW

File: build-system/compile/compile.js [hunk 1]
```diff
@@ -359,12 +352,6 @@ function compile(
           )
         )
         .on('error', reject)
-        .pipe(
-          gulpIf(
-            shouldAppendSourcemappingURLText,
-            gap.appendText(`\n//# sourceMappingURL=${outputFilename}.map`)
```
Comment (rsimha) [inline build-system/compile/compile.js:365]: With this, we can uninstall `gulp-append-prepend` from `package.json`

**Review by rsimha — APPROVED** [review]
LGTM with one comment.

**Review by kristoferbaxter — APPROVED** [review]

File: build-system/compile/compile.js (hunk 1 above)
Comment (jridgewell) [reply build-system/compile/compile.js:408]: Done.

**Comment by jridgewell** [comment]
@samouri: Can I get a bundle size approval?

**Review by kristoferbaxter — APPROVED** [review]

**Pull request merged**